src/controllers/ScheduleController.ts

```diff
@@ -13,10 +14,7 @@ export class ScheduleController {
 
   public async schedule(req: Request, res: Response) {
     try {
-      const stored = await this.scheduleService.schedule(
-        req.body,
-        req.query as any,
-      );
+      const stored = await this.scheduleService.schedule(req.body, normalizeScheduleRequestParams(req.query));
       res.json({
         id: stored._id,
```

Comment:
here id doesnt have a type, maybe we can define a type for it?

Reply:
It kinda does, since it's the same type as `stored._id` and that one is actually a mongodb id (i.e. string). but it's true that we could define some return types for the whole response. however, express does not let you do type arguments for json response (`res.json<type>({ ... })`) so it's let's leave it like that for now